crates/biome_js_analyze/src/lint/correctness/no_undeclared_dependencies.rs

```diff
@@ -64,6 +64,8 @@ impl Rule for NoUndeclaredDependencies {
             // TODO: we should also check that an `.` exports exists.
             // See https://nodejs.org/api/packages.html#self-referencing-a-package-using-its-name
             || ctx.name() == Some(package_name)
+            // ignore NodeJS builtin modules
```

Comment:
```suggestion
            // ignore Node.js builtin modules
```

FYI `Node.js` is the correct and official name